src/Microsoft.Fx.Portability/IAssemblyFile.cs

```diff
@@ -0,0 +1,16 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System.IO;
+
+namespace Microsoft.Fx.Portability
+{
+    public interface IAssemblyFile
+    {
+        Stream OpenRead();
+
+        string Path { get; }
```

Comment:
Cool. The places I'm thinking of that could be problematic are, for example, `MetadataReaderExtensions.GetAssemblyInfo` [here](https://github.com/Microsoft/dotnet-apiport/blob/master/src/Microsoft.Fx.Portability.MetadataReader/MetadataReaderExtensions.cs)